src/cpp/src/continuous_batching/pipeline.cpp

```diff
@@ -279,6 +279,7 @@ std::vector<VLMDecodedResults> ContinuousBatchingPipeline::generate(
 
 
 void ContinuousBatchingPipeline::start_chat(const std::string& system_message) {
+    m_impl->finish_chat();
```

Comment:
can we test it? as i understand, you just need to check output from the second chat.

Reply:
Yes, done.